fdbrpc/sim2.actor.cpp

```diff
@@ -1756,6 +1756,11 @@ Future< Void > Sim2FileSystem::deleteFile( std::string filename, bool mustBeDura
 	return Sim2::deleteFileImpl(&g_sim2, filename, mustBeDurable);
 }
 
+Future< std::time_t > Sim2FileSystem::lastWriteTime( std::string filename ) {
+	// TODO(alexmiller): FIXME
```

Comment:
This should be implemented so that this monitoring / using new certificates logic is tested in simulation.  An easy implementation would be to have a function-local static map of filename to last write time, with each one starting at 0 and then update it to and return now() randomly, maybe 1% of the time.

Reply:
To record the in-person conversation, it'd be better to actually maintain a map that's updated whenever we write to a file in simulation.  However, the only use we have of lastWriteTime is for files we never write from simulation, so this change would be A Good Thing to do, but not actually useful at all right now.

So I've done exactly what you proposed.